app/labor/markdown_parser.rb

```diff
@@ -174,6 +177,34 @@ def escape_liquid_tags_in_codeblock(content)
     end
   end
 
+  def possibly_raw_tag_syntax?(array)
+    array.each do |string|
```

Comment:
This can be simplified by using `any?`:

```ruby
[8] pry(main)> ["abc", "raw"].any? {|string| ["{", "}", "raw", "endraw", "----"].include?(string) }
=> true
[9] pry(main)> ["abc", "def"].any? {|string| ["{", "}", "raw", "endraw", "----"].include?(string) }
=> false
```